docs/guides/virtualization/cloud-init/02_first_contact.md

```diff
@@ -0,0 +1,151 @@
+---
+Title: II. First Contact: Simple Bootstrapping
+author: Wale Soyinka
+contributors:
+tags:
+  - cloud-init
+  - cloud
+  - automation
+---
+
+
+# **II. First Contact: Simple Bootstrapping with the Rocky Linux 10 QCOW2 Image**
+
+In the previous chapter, we covered the fundamental concepts of `cloud-init`. Now, it’s time to move from theory to practice. This chapter is your first hands-on mission: you will take the official Rocky Linux 10 Generic Cloud image, provide it with a simple set of instructions, and watch it configure itself on first boot.
+
+## **1. Prepping the Lab Environment**
+
+Before we can boot our first instance, we need to prepare our local lab environment. For these exercises, we will simulate a cloud environment using standard Linux virtualization tools.
+
+### **Prerequisites: Host Tools**
+
+Ensure you have the following tools installed on your host machine. On a Rocky Linux host, you can install them with `dnf`:
+
+```bash
+sudo dnf install -y libvirt qemu-kvm virt-install genisoimage
+```
+
+*   **Virtualization Hypervisor:** A tool like KVM/QEMU or VirtualBox.
+*   `virt-install`: A command-line utility for provisioning new virtual machines.
+*   `genisoimage` (or `mkisofs`): A tool to create an ISO9660 filesystem.
+
+### **The QCOW2 Image**
+
+If you haven't already, download the official Rocky Linux 10 Generic Cloud image.
+
+```bash
+curl -L -o Rocky-10-GenericCloud.qcow2 \
+https://dl.rockylinux.org/pub/rocky/10/images/x86_64/Rocky-10-GenericCloud-Base.latest.x86_64.qcow2
+```
+
+To preserve the original, create a working copy of the image for your VM.
+
+```bash
+cp Rocky-10-GenericCloud.qcow2 first-boot-vm.qcow2
+```
+
+!!! tip "Save Disk Space with Backing Files"
+    A full copy of the image can be large. To save disk space, you can create a *linked clone* that uses the original image as a backing file. This creates a much smaller `qcow2` file that only stores the differences from the original.
+
```

Comment:
You will need an extra LF after the admonition title line for proper formatting:

```
!!! tip "Save disk space with backing files"

    A full copy....
    
```